File: Android/sources/androidx/media3/datasource/HttpDataSource$InvalidResponseCodeException.java
package androidx.media3.datasource;

import java.io.IOException;
import java.util.Map;

public final class HttpDataSource$InvalidResponseCodeException extends HttpDataSource$HttpDataSourceException {
    public final int o;
    public final Map v;

    public HttpDataSource$InvalidResponseCodeException(int i, DataSourceException dataSourceException, Map map) {
        super(wj6.h(i, "Response code: "), (IOException) dataSourceException, 2004);
        this.o = i;
        this.v = map;
    }
}

File: IOS/v2/iosx/media3/datasource/HttpDataSource$InvalidResponseCodeException.swift
package iOSx.media3.datasource;.swift
import java.io.IOException;.swift
import java.util.Map;.swift
public final class HttpDataSource$InvalidResponseCodeException extends HttpDataSource$HttpDataSourceException {.swift
public final int o;.swift
public final Map v;.swift
public HttpDataSource$InvalidResponseCodeException(int i, DataSourceException dataSourceException, Map map) {.swift
super(wj6.h(i, "Response code: "), (IOException) dataSourceException, 2004);.swift
this.o = i;.swift
this.v = map;.swift
}.swift
}.swift
